test/plots/crimean-war-arrow.js

```diff
@@ -0,0 +1,18 @@
+import * as Plot from "@observablehq/plot";
+import * as d3 from "d3";
+
+export default async function() {
+  const crimea = await d3.csv("data/crimean-war.csv", d3.autoType);
+  const causes = crimea.columns.slice(2);
+  const data = causes.flatMap(cause => crimea.map(({date, [cause]: deaths}) => ({date, cause, deaths})));
+  return Plot.plot({
+    x: {
+      tickFormat: "%b",
+      label: null
+    },
+    marks: [
+      Plot.ruleY([0]),
+      Plot.lineY(data, {x: "date", y: "deaths", stroke: "cause", marker: "arrow"})
```

Comment:
> I wonder if we must check what a custom marker returns—either verify that it is a svg:marker, or at least that it is not nullish? I've documented it as a MUST, but if we accepted nullish values it would allow to not return a marker for some colors. Currently one has to return an empty svg:marker—which is probably fine too.

We could maybe add this in the future, but I don’t think we need to do it now. I would add it if it’s frequently needed (for convenience), or is a frequent source of errors (for debugging). For now I think implementing a custom marker function is a relative “power user” feature, similar to a custom mark implementation, so it can have sharper edges than the more commonly-used APIs.

Reply:
> this version feels more natural

I find the natural curve tends to be a little… loosey-goosey, like where it dips below _y_ = 0; catmull-rom tends to do better, and monotone-x makes a hard guarantee. Think I’ll just keep it simple for now though.